webapp/sources/rudder/rudder-web/src/main/javascript/rudder/tree.js

```diff
@@ -264,32 +287,7 @@ var buildGroupTree = function(id, appContext, initially_select, select_multiple_
   var allNodeEvents = allNodes.map(function () {
     return $._data(this, 'events')["click"]; // we need to intercept the click now, the object will change after the jstree init
   }).get().map(function (x, i) { return [allNodes[i], x] });  // cannot use a node object as key, so keep a [node, events] array
-
-  (function ($, undefined) {
-    "use strict";
-    $.jstree.defaults.keepevent = $.noop;
-    $.jstree.plugins.keepevent = function (options, parent) {
-      this.redraw_node = function (obj, deep, callback, force_draw) {
-        obj = parent.redraw_node.call(this, obj, deep, callback, force_draw);
-        if (obj) {
-          var child = $(obj).find('[id^="lift-event-js-"]'); // all nested child elements with a 'lift-event-js-...' id
```

Comment:
This select also the inner childrens, like groups or directives, and bound several times the click event

Reply:
(in fact replaced unused click.jstree event)